src/postamble_minimal.js

```diff
@@ -47,10 +47,12 @@ function initRuntime(asm) {
 // Initialize wasm (asynchronous)
 
 var imports = {
+#if MINIFY_WASM_IMPORTED_MODULES
+  'a': asmLibraryArg,
```

Comment:
Hmm, user code needs to specify this, it doesn't occur automatically? Or I suppose there will always be just one canonical import by the name `a` so no point doing an automatic change?

Reply:
@juj not sure what you mean? This optimization happens automatically when optimizing at higher levels (-O3, -Os).

Yes, when this optimization happens, there is always a single imported module of name `"a"`.

Reply:
Gotcha. Yeah, I was wondering that this change is assisted here and not automatically transformed by the optimizer - but doesn't make sense to write code for that transform given it always transforms to just one var `a` after all.